src/_hooks/useValidateSchema.ts

```diff
@@ -8,7 +9,10 @@ export function useValidateSchema<T>(
   schema?: yup.Schema<T>,
   value?: T,
   { abortEarly, recursive }: yup.ValidateOptions = {},
-): string[] {
+) {
+  const [debouncedValue] = useDebounce(value, 500);
```

Comment:
Maybe we can pass the debounce timer in as an option to FormInput?

```suggestion
  const [debouncedValue] = useDebounce(value, debounce);
```